src/trie.py

```diff
@@ -0,0 +1,93 @@
+"""Module for Trie tree."""
+
+# TRIE TREE (TT)
+#
+# CodeFellows 401d5
+# Submission Date:
+#
+# Authors:  Colin Lamont <https://github.com/chamberi>
+#           Ben Shields <https://github.com/iamrobinhood12345>
+#
+# URL:
+
+
+class Node(object):
+    """Node Class contains a value and a dictionary of nodes."""
+
+    def __init__(self, val=None):
+        """Initialize the Node class with val and empty nodes dictionary."""
+        self.val = val
+        self.nodes = {}
+
+
+class Trie(object):
+    """Trie class, which is the Trie tree."""
+
+    """
```

Comment:
not necessary.  Your comments should look like this:
"""Trie class, which is the Trie tree.

insert blah blah
contains blah blah
"""